RELEASE.md

```diff
@@ -14,7 +14,48 @@ Further details on building, testing and use can be found in [README.md](https:/
 
 ## Release notes
 
-This is version 0.7.1-dev of oqs-provider which continues from the earlier 0.7.0 release. This release is fully tested to be used in conjunction with the main branch of [liboqs](https://github.com/open-quantum-safe/liboqs) and is guaranteed to be in sync with v0.12.0 of `liboqs` as and when released.
+This is version 0.8.0-rc1 of oqs-provider which continues from the earlier 0.7.0 release. This release is fully tested to be used in conjunction with the main branch of [liboqs](https://github.com/open-quantum-safe/liboqs) and is guaranteed to be in sync with v0.12.0 of `liboqs`.
+
+### Deprecation notice
+
+This is to notify users of Kyber (Round 3 version) to switch to the ML-KEM (FIPS 203 final version) as support for Kyber will be removed with the next release of liboqs and oqsprovider.
+
+The addition of ML-DSA FIPS 204 final version to liboqs v0.12.0 has introduced a new signature API which includes a context string parameter. The liboqs team is planning to remove the old version of the API without a context string in the next release to streamline the API and bring it in line with NIST specifications. Users who have an opinion on this removal are invited to provide input at [liboqs #2001](https://github.com/open-quantum-safe/liboqs/issues/2001).
+
+### Security considerations
+
+* CVE-2024-54137: The associated liboqs v0.12.0 release fixed a bug in HQC decapsulation that leads to incorrect shared secret value during decapsulation when called with an invalid ciphertext. Thank you to Célian Glénaz and Dahmun Goudarzi from Quarkslab for identifying the issue.
+
+### What's New
+
+In addition to improving testing, CI, and fixing platform specific build issues this release of oqs-provider:
+
+* Updates IANA code points for ML-KEM.
```

Comment:
Well, also those for Frodo, obviously. Although those are not IANA code points, strictly speaking :)

Reply:
Thank you pointing this out @baentsch! I copied the PR title over from #577 without inspecting file level changes. Could you please suggest a more suitable description?

Reply:
What about "updates IANA code points for ML-KEM also changing Frodo code points"? The latter are not IANA approved but at least now are in a reserved space.